TodoList.Web/Controllers/TodosController.cs

```diff
@@ -34,15 +34,23 @@ public async Task<IActionResult> Home()
             var currentUser = await _userManager.GetUserAsync(User);
             if (currentUser == null) return Challenge();
 
+            var now = DateTime.Now;
```

Comment:
You can omit that variable and use `DateTime.Now` in each occurrence. It is used only tow times.

Reply:
Will do.